rdfunit-junit/README.md

```diff
@@ -0,0 +1,40 @@
+RDFUnit - JUnit Integration
+==========
+
+RDFUnit-JUnit Integration provides an interface to JUnit via a dedicated Runner which exposes single RDFUnit Test Cases as Unit-Tests.
+
+### Usage
+```java
+
+@RunWith(RdfUnitJunitRunner.class)
+@Schema(uri = "ontologies/foaf.rdf")
+public static class TestRunner {
+
+    @ControlledVocabulary
```

Comment:
Good Point - 'ControlledVocabulary' is just our use case (as we're loading CVs as additional data) but in general terms `@AdditionalData` is more suited.

In fact it gets merged with each InputModel later, but we need to keep both separated to figure out if the offending resources found during test execution is actually part of the InputModel or not.